meta-narwhal/recipes-core/hands-timesync/hands-timesync.bb

```diff
@@ -0,0 +1,22 @@
+DESCRIPTION = "Utility to set correct time on LG Watch W7"
+HOMEPAGE = "https://github.com/AsteroidOS/narwhal-hands-timesync.git"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=84dcc94da3adb52b53ae4fa38fe49e5d"
+
+SRC_URI = "git://github.com/asteroidos/narwhal-hands-timesync.git;branch=main;protocol=https"
+SRCREV = "${AUTOREV}"
+PR = "r0"
+PV = "+git${SRCPV}"
+S = "${WORKDIR}/git"
+DEPENDS = "qtbase"
+inherit cmake_qt5 pkgconfig
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+do_install:append() {
+    install -d ${D}/usr/lib/systemd/system/
```

Comment:
Isn't this path created by CMake?

Reply:
it should be. I was working on the basis of lcd-tools, and I stuck with the format used there. will remove.